docs/services/zwo_camera.rst

```diff
@@ -1,14 +1,64 @@
 ZWO Camera
```

Comment:
It needs to be stated here that using the ZWO cameras requires the installation of their drivers - you can copy that information from here, and delete the comment from the env file:
https://github.com/spacetelescope/catkit2/blob/0670c9efa3f764edafbc4b33c3f4c6a125efe18a/environment.yml#L41